sorbet/rbi/shims/active_model.rbi

```diff
@@ -0,0 +1,13 @@
+# typed: true
```

Comment:
Nitpick: Can we rename this file to be `activemodel.rbi` for consistency with the gem name.